src/components/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({ title, date, text }) {
```

Comment:
if you have nothing written before return, you can use an arrow function () => ()